ID.me WebVerify SDK/IDmeWebVerifyKeychainData.h

```diff
@@ -13,12 +13,14 @@
 
 @interface IDmeWebVerifyKeychainData : NSObject
 
-@property (nonatomic, strong) NSString *accessToken;
-@property (nonatomic, strong) NSString *refreshToken;
-@property (nonatomic, strong) NSDate *expirationDate;
-@property (nonatomic, strong) NSString *scope;
+-(NSString* _Nullable)getLatestUsedScope;
+-(NSString* _Nullable)accessTokenForScope:(NSString* _Nonnull)scope;
+-(NSDate* _Nullable)expirationDateForScope:(NSString* _Nonnull)scope;
+-(NSString* _Nullable)refreshTokenForScope:(NSString* _Nonnull)scope;
+
+-(void)setToken:(NSString * _Nonnull)accessToken expirationDate:(NSDate * _Nonnull)date
+   refreshToken:(NSString * _Nullable)refreshToken forScope:(NSString * _Nonnull)scope;
```

Comment:
It is nullable as it will not be updated when the access token gets updated so passing nil will leave the old refresh token. We can change this if it is not clear. Or we should document it